woocommerce/woocommerce
# Display ratings on marketplace product cards

### Changes proposed in this Pull Request:

Closes https://github.com/Automattic/woocommerce.com/issues/18961

This PR displays ratings on the product card in the In-app marketplace. The ratings already exist in the API, they just needed displaying in the component. This PR:

- Adds the ratings to all product cards
- Updates the theme card styling to fit with the original design
- Fixes a couple of minor CSS bugs in product cards



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch and build (`pnpm --filter="@woocommerce/plugin-woocommerce" build`)
2. Load WooCommerce > Extensions ([link](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions))
3. See that there are ratings on products and no products showing with 0 ratings (these should be hidden)
4. Check theme cards to ensure that the price (and ratings, although there aren't many themes with ratings to check) is displaying below the theme and vendor name.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Display review ratings on Extension marketplace product cards

#### Comment 

</details>
